apps/member/src/pages/SupportListPage/components/SupportTableRow/SupportTableRow.tsx

```diff
@@ -0,0 +1,66 @@
+import { Table } from '@clab-platforms/design-system';
+import { cn } from '@clab-platforms/utils';
+
+import { useMyProfile, useSupportDetail } from '@hooks/queries';
+import { formattedDate, toKoreaISOString } from '@utils/date';
+import { formatMemberName } from '@utils/string';
+
+import { Support } from '@type/support';
```

Comment:
```suggestion
import type { Support } from '@type/support';
```
type을 가져오는 경우에도 type import가 적용되어 있지 않은 파일이 많네요. 진규님도 수정하시면서 발견하면 type import 추가해주시면 좋을 것 같아요. 이거 이후로 일일이 표시하지는 않겠습니다!